code/datums/ruins/space_ruins.dm

```diff
@@ -212,8 +212,8 @@
 /datum/map_template/ruin/space/dj
 	id = "dj"
 	suffix = "dj.dmm"
-	name = "Russian DJ Station"
-	description = "An old russian listening station, long since defunct and lifeless, however the equipment is likely still in working condition."
+	name = "Soviet DJ Station"
+	description = "An old Union listening station, long since defunct and lifeless, however the equipment is likely still in working condition."
```

Comment:
```suggestion
	description = "An old soviet era listening station, long since defunct and lifeless, however the equipment is likely still in working condition."
```